src/01_bignum.py

```diff
@@ -1,4 +1,9 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+print(2 ** 25536)
```

Comment:
What happens in the JS console?

Reply:
Oh, I forgot to write it there but it shows Inifinity in JS console.